src/app/profile/overview/overview-routing.module.ts

```diff
@@ -2,11 +2,27 @@ import { NgModule }  from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import { OverviewComponent } from './overview.component';
+import { SpacesComponent } from './spaces/spaces.component';
+import { WorkItemsComponent } from './work-items/work-items.component';
 
 const routes: Routes = [
   {
     path: '',
-    component: OverviewComponent
+    component: OverviewComponent,
+    children: [
+      {
+        path: '',
+        redirectTo: '_workitems'
+      },
+      {
+        path: '_workitems',
+        component: WorkItemsComponent
+      },
+      {
+        path: '_spaces',
```

Comment:
@andrewazores This path was already defined for the spaces.component [0], so clicking "View all spaces" from the header [1] will now route to the profile page my-spaces widget, I'm not sure if this was your intent or not but it renders the "View all spaces" page unreachable (which I don't think is the biggest deal because it's supposed to be removed anyways [2]) but thought I'd just confirm.

[0] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/profile/profile-routing.module.ts#L21
[1] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/layout/header/header.component.html#L101
[2] https://github.com/openshiftio/openshift.io/issues/3536#issuecomment-390823742

Reply:
Ah, no, that was not intentional. I intended for that to be a unique path. I'll raise a PR momentarily to fix this path collision so the old "View All Spaces" page is reachable, until UXD gives guidance on what should be done with it instead.